--- a/app/Http/Controllers/IndexController.php
+++ b/app/Http/Controllers/IndexController.php
@@ -3,12 +3,14 @@
 namespace App\Http\Controllers;

 use Inertia\Inertia;
+use Modules\Billing\Models\Product;

 class IndexController extends Controller
 {
     public function __invoke()
     {
         return Inertia::render('Index', [
             // Share here your frontend data, e.g. products, announcements, etc.
+            'products' => Product::displayable()->get(),
         ])->withSSR();
     }
 }
